### PR TITLE
feat: disable london ticket button

### PR DESCRIPTION
**Pull request by thulieblack**

### CODE REVIEW

**Review by AceTheCreator — CHANGES_REQUESTED** [review]
I don't think you should remove the ticket url. Just changing the ended value will do :)

File: config/city-lists.json [hunk 1]
```diff
@@ -45,9 +45,9 @@
 		"sponsors": [
 			"/img/apidays.png"
 		],
-		"ticket": "https://ticket.apidays.global/event/apidays-london-2024/869eca20-bfb6-4103-b8bb-d0348932e940/cart?coupon=GUESTOFASYNCAPILONDON2024",
+		"ticket": false,
```
Comment (thulieblack) [inline config/city-lists.json:48]:
```suggestion
		"ticket": "https://ticket.apidays.global/event/apidays-london-2024/869eca20-bfb6-4103-b8bb-d0348932e940/",
```

**Comment by thulieblack** [comment]
tickets sold out has to merge this asap

**Pull request merged**